tasks/main.yml

```diff
@@ -1,9 +1,9 @@
 ---
 # tasks file for sa-hashicorp-vault
-  - include: tasks_nginx.yml
+  - include_tasks: tasks_nginx.yml
     when: option_install_nginx
 
-  - include: tasks_vault.yml
+  - import_tasks: tasks_vault.yml
 
-  - include: vault_console_tools.yml
+  - include_tasks: vault_console_tools.yml
```

Comment:
To be honest, the most strangest update in ansible 2.4
`Include` was clear enough